Documentation/proposals/OneDotNetMSBuildProperties.md

```diff
@@ -0,0 +1,178 @@
+# One .NET: New MSBuild Properties
```

Comment:
Should this title include "Single App" or "Unified" or some other noun to suggest what's special about *these* properties vs. all the other properties we already support?

Reply:
@Redth can we use the term "Single Project" or probably not?